datasets/bnl_newspapers/README.md

```diff
@@ -0,0 +1,206 @@
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:**: [https://data.bnl.lu/data/historical-newspapers/](https://data.bnl.lu/data/historical-newspapers/) 
+- **Repository:**
+- **Paper:**
+- **Leaderboard:**
+- **Point of Contact:**
+
+### Dataset Summary
+
+The BnL has digitised over 800.000 pages of Luxembourg newspapers. This dataset contains a subset of these newspapers which sit under the "Processed Datasets" collection. The BNL:
+
+> processed all newspapers and monographs that are in the public domain and extracted the full text and associated meta data of every single article, section, advertisement… The result is a large number of small, easy to use XML files formatted using Dublin Core.
+
+This dataset contains the newspapers from these processed collections. 
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+The dataset currently contains a single configuration.
+
+### Data Instances
+
+An example instance from the datasets:
+
+``` python
+{'ark_id': 'https://persist.lu/ark:/70795/wx8r4c/articles/DTL47',
+ 'article_type': 8,
+ 'extent': 49,
+ 'ispartof': 'Luxemburger Wort',
+ 'pub_date': datetime.datetime(1853, 3, 23, 0, 0),
+ 'publisher': 'Verl. der St-Paulus-Druckerei',
+ 'source': 'newspaper/luxwort/1853-03-23',
+ 'text': 'Asien. Eine neue Nedcrland-Post ist angekommen mil Nachrichten aus Calcutta bis zum 5. Febr.; Vom» vay, 12. Febr. ; Nangun und HongKong, 13. Jan. Die durch die letzte Post gebrachle Nachricht, der König von Ava sei durch seinen Bruder enlhronl worden, wird bestätigt. (K. Z.) Verantwortl. Herausgeber, F. Schümann.',
+ 'title': 'Asien.',
+ 'url': 'http://www.eluxemburgensia.lu/webclient/DeliveryManager?pid=209701#panel:pp|issue:209701|article:DTL47'}
+```
+
+
+### Data Fields
+
+- 'ark_id': This is a unique and persistent identifier using ARK. 
```

Comment:
This should be made consistent:
- either renaming it to "id"
- or renaming back the feature to "ark_id"